### PR TITLE
Add cppwinrt and cswinrt tests in windowsai nuget pipeline

### PR DESCRIPTION
**Pull request by smk2007**
Add cppwinrt and cswinrt tests in windowsai nuget pipeline

Issue: No E2E tests that validate the contents of the Microsoft.AI.MachineLearning.nupkg pipeline exist. 

Fix: Add a few simple tests cases that restore the Microsoft.AI.MachineLearning.nupkg from the build and evaluate squeezenet e2e for cppwinrt and cswinrt.

### CODE REVIEW

File: csharp/test/Microsoft.AI.MachineLearning.Tests.CSharp/Microsoft.AI.MachineLearning.Tests.CSharp.csproj.pp [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'Debug|x64'">
+    <DebugSymbols>true</DebugSymbols>
+    <OutputPath>bin\x64\Debug\</OutputPath>
+    <DefineConstants>DEBUG;TRACE</DefineConstants>
+    <DebugType>full</DebugType>
+    <PlatformTarget>x64</PlatformTarget>
+    <LangVersion>7.3</LangVersion>
+    <ErrorReport>prompt</ErrorReport>
+    <CodeAnalysisRuleSet>MinimumRecommendedRules.ruleset</CodeAnalysisRuleSet>
+    <Prefer32Bit>true</Prefer32Bit>
```
Comment (tiagoshibata) [inline csharp/test/Microsoft.AI.MachineLearning.Tests.CSharp/Microsoft.AI.MachineLearning.Tests.CSharp.csproj.pp:47]: Why?

Comment (tiagoshibata) [reply csharp/test/Microsoft.AI.MachineLearning.Tests.CSharp/Microsoft.AI.MachineLearning.Tests.CSharp.csproj.pp:47]: The flag is only valid in AnyCPU builds

Comment (smk2007) [reply csharp/test/Microsoft.AI.MachineLearning.Tests.CSharp/Microsoft.AI.MachineLearning.Tests.CSharp.csproj.pp:47]: default visual studio properties.

File: csharp/test/Microsoft.AI.MachineLearning.Tests.CSharp/main.cs [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+               var image_task = Windows.Storage.StorageFile.GetFileFromPathAsync(name);
+               image_task.AsTask().Wait();
+               var image = image_task.GetResults();
+               Console.WriteLine("Load StorageFile into Stream.");
+               var stream_task = image.OpenReadAsync();
+               System.Threading.Thread.Sleep(1000);
+               // stream_task.AsTask().Wait();
+               //
+               // Unable to call AsTask on IAsyncOperation<IRandomAccessStreamWithContentType>... 
+               // System.TypeInitializationException: 'The type initializer for 'ABI.Windows.Foundation.AsyncOperationCompletedHandler`1' threw an exception.'
```
Comment (ryanlai2) [inline csharp/test/Microsoft.AI.MachineLearning.Tests.CSharp/main.cs:27]: Is there any work for us to investigate why this happens? Is this a bug in WinML?

Comment (smk2007) [reply csharp/test/Microsoft.AI.MachineLearning.Tests.CSharp/main.cs:27]: Its a bug in cswinrt. They are fixing it.

**Pull request merged**